src/lib/Scenes/City/City.tsx

```diff
@@ -123,62 +132,63 @@ export class CityView extends Component<Props, State> {
   }
 
   render() {
-    const { buckets, filter, cityName, citySlug } = this.state
+    const { buckets, cityName, citySlug } = this.state
     const { verticalMargin } = this.props
     // bottomInset is used for the ScrollView's contentInset. See the note in ARMapContainerViewController.m for context.
     const bottomInset = this.scrollViewVerticalStart + (verticalMargin || 0)
+
     return (
       buckets && (
         <Theme>
           <>
-            <Box>
+            <Flex style={{ flex: 1 }}>
               <Flex py={1} alignItems="center">
                 <Handle />
               </Flex>
-              <FiltersBar
-                tabs={cityTabs}
-                goToPage={activeIndex => EventEmitter.dispatch("filters:change", activeIndex)}
-              />
-              <ScrollView
-                contentInset={{ bottom: bottomInset }}
-                onLayout={layout => {
-                  this.scrollViewVerticalStart = layout.nativeEvent.layout.y
-                  NativeModules.ARNotificationsManager.postNotificationName("ARLocalDiscoveryCityGotScrollView", {})
-                }}
-                ref={r => {
-                  if (r) {
-                    this.scrollView = r as any
-                  }
+              <ScrollableTabView
+                initialPage={this.props.initialTab || AllCityMetaTab}
+                onChangeTab={selectedTab => this.setSelectedTab(selectedTab)}
+                renderTabBar={props => (
+                  <>
+                    <TabBar {...props} />
```

Comment:
Can we use the same tab bar as the Saves & Follows VC https://github.com/artsy/emission/blob/master/src/lib/Scenes/Favorites/index.tsx#L65